src/components/modeler/Modeler.vue

```diff
@@ -121,12 +126,7 @@ import PaperManager from '../paperManager';
 import registerInspectorExtension from '@/components/InspectorExtensionManager';
 
 import initAnchor from '@/mixins/linkManager.js';
-import {
-  addIdToNodeAndSetUpDiagramReference,
-  addNodeToProcess,
-  getTargetProcess,
-  isBoundaryEvent,
-} from '@/components/nodeManager';
+import { addIdToNodeAndSetUpDiagramReference, addNodeToProcess, getTargetProcess, isBoundaryEvent } from '@/components/nodeManager';
```

Comment:
You might want to rebase from develop: some of these have been deleted and brought into the new Node class.